packages/x-date-pickers-pro/src/DesktopDateRangePicker/tests/DesktopDateRangePicker.test.tsx

```diff
@@ -488,65 +521,10 @@ describe('<DesktopDateRangePicker />', () => {
       expect(onClose.callCount).to.equal(1);
     });
 
-    it('should call onClose, onChange with empty value and onAccept with empty value when pressing the "Clear" button', () => {
```

Comment:
All those tests (and the equivalent on `MobileDateRangePicker.test.tsx`) are already handled by `testPickerActionBar.tsx`